pkgs/esp-idf/default.nix

```diff
@@ -1,5 +1,5 @@
 { rev ? "v5.1.2"
-, sha256 ? "sha256-uEf3/3NPH+E39VgQ02AbxTG7nmG5bQlhwk/WcTeAUfg="
+, sha256 ? "sha256-qjZ438LUQLzyQDKkeo4WMO0jxfkFh6dReuLq/6rftpw="
```

Comment:
Is this change intentional?  I'm trying to test using your fork but this appears to be breaking it.

Reply:
Hmm, it was working when I made the change. If I rebuild now it also stopped working for me. I will update the hash, and we will see if this fixes the issue for now.